api/config/config_test.json

```diff
@@ -185,6 +185,6 @@
     "latestAouVersion": 1
   },
   "tanagra": {
-    "baseUrl": "https:\/\/aou-tanagra-dev.fake-research-aou.org"
+    "baseUrl": "https:\/\/aou-tanagra.dev.pmi-ops.org/"
```

Comment:
not sure you need the slash on the end.

Reply:
Good catch. Copied from the browser and didn't notice that